projects/rocprofiler-systems/source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -646,6 +994,12 @@ tool_tracing_callback(rocprofiler_callback_tracing_record_t record,
                                     record.kind);
                 break;
             }
+            default:
+            {
+                ROCPROFSYS_CI_ABORT(true, "Unhandled callback record kind: %i\n",
+                                    record.kind);
```

Comment:
```suggestion
                ROCPROFSYS_CI_ABORT(true, "Unhandled callback record.\n\t%s\n",
                                    info.str().c_str());
```
Maybe we can use the info string for these error messages now?